editor/src/components/canvas/canvas-floating-toolbars.tsx

```diff
@@ -43,6 +44,11 @@ export const CanvasFloatingToolbars = React.memo((props: { style: React.CSSPrope
           <CanvasStrategyPicker />
         </FlexRow>
       </FlexRow>
+      <FlexRow style={{ marginTop: `auto` }}>
+        <FlexRow style={{ marginBottom: '6.5px' }}>
```

Comment:
Can these be squashed into a single `FlexRow`, or does that actually mess up the style?

Reply:
I don't believe so, I tried using `calc-size` but that didn't work at all.

Reply:
btw, I tried a single `FlexRow` with `style: {{ marginTop: 'auto', marginBottom: '6.5px' }}` and it seems like it looks the same... unless there is another special case where they are differ?
this is with this toast:
```tsx
<React.Fragment>
  <div>this is a test</div>
  <div>this is another test</div>
</React.Fragment>
```
<img width="418" alt="image" src="https://github.com/user-attachments/assets/c50b944c-4a6f-42f7-b568-3c215ea5c168">

Reply:
I stand corrected, applied this change in 933bccd4d.